packages/search/tests/integration/components/navi-search-result/definition-test.js

```diff
@@ -127,4 +119,119 @@ module('Integration | Component | definition', function(hooks) {
       'definitions sources are shown correctly'
     );
   });
+
+  test('result without a description', async function(assert) {
+    assert.expect(1);
+
+    const data = [
+      {
+        id: 'pageViews',
+        name: 'Page Views'
+      }
+    ];
+    set(this, 'data', data);
+
+    await render(hbs`<NaviSearchResult::Definition @data={{this.data}} />`);
+    assert.dom('.navi-search-definition-result').doesNotExist('No results are displayed');
+  });
+
+  test('show more results', async function(assert) {
+    assert.expect(12);
+    const data = [
+      {
+        id: 'pageViews',
+        name: 'Page Views',
+        description: 'The number of views of a page.'
+      },
+      {
+        id: 'impressions',
+        name: 'Impressions',
+        description: 'Number of times a user saw the ad.'
+      },
+      {
+        id: 'revenue',
+        name: 'Revenue',
+        description: 'How much money were made.'
+      }
+    ];
+    set(this, 'data', data);
+
+    await render(hbs`<NaviSearchResult::Definition @data={{this.data}} />`);
+
+    assert.dom('.navi-search-definition-result').exists({ count: 2 }, 'Two results are displayed');
+    assert.equal(
+      find('.navi-search-result-options__button').textContent.trim(),
+      'Show more',
+      'Show less button is shown.'
+    );
+
+    assert.deepEqual(
+      findAll('.navi-search-definition-result__item-name').map(el => el.textContent.trim()),
+      ['Page Views', 'Impressions'],
+      'definition titles are shown correctly'
+    );
+    assert.deepEqual(
+      findAll('.navi-search-definition-result__item-description').map(el => el.textContent.trim()),
+      ['The number of views of a page.', 'Number of times a user saw the ad.'],
+      'definition descriptions are shown correctly'
+    );
+
+    await click('.navi-search-result-options__button');
```

Comment:
element class name is a bit ambiguous. Could it be `navi-search-result-options__show-button`?